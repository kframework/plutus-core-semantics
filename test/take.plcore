(program

(module Foo (imported) (exported () ()) (local () ())
(define b1 (builtin takeByteString 1 #123456))

(define b2 (builtin takeByteString 2 #123456))

(define b3 (builtin takeByteString 3 #123456))
)

)
