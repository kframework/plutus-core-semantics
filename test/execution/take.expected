<T>
  <k>
    .
  </k>
  <lenv>
    .Map
  </lenv>
  <genv>
    Foo . b1 |-> #1
    Foo . b2 |-> #12
    Foo . b3 |-> #123
  </genv>
  <modules>
    ModuleCellSetItem ( <Module>
      <curr>
        0
      </curr>
      <name>
        Foo
      </name>
      <dataCons>
        .TyExpList
      </dataCons>
      <decl>
        .QualCList
      </decl>
      <importedModules>
        .ModNmList
      </importedModules>
      <exportTy>
        .TyExpList
      </exportTy>
      <exportTm>
        .QualCList
      </exportTm>
    </Module> )
  </modules>
</T>
